components/shared/SocialIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "next/image";
+interface IconProps {
+  name: string;
+  width?: number;
+  height?: number;
+  className?: string;
+}
+const SocialIcon = ({ name, width, height, className }: IconProps) => {
+  const iconPath = `/icons/${name}.svg`;
+  return (
+    <Image
+      src={iconPath}
+      alt={name}
+      width={width}
+      height={height}
+      className={className}
+    />
+  );
+};
```

Comment:
There's a little confusion here with your imports. Now that you're sending the images down directly- you don't need to generate an iconPath, you would send the image. I don't think we need to make so many Components here though. We're three components deep for a button.

I think we can remove socialIcon altogether. Let's look at the Social Auth Group and SocialAuthButton to fix this.

Reply:
yeah, I had the same idea,I was kind of rushing to catch up - but when I get back to this , I'll follow the guidance